barretenberg/cpp/src/barretenberg/vm2/constraining/relations/data_copy.test.cpp

```diff
@@ -36,19 +37,19 @@ class DataCopyConstrainingBuilderTest : public ::testing::Test {
   protected:
     DataCopyConstrainingBuilderTest()
     {
-        ON_CALL(context, get_memory).WillByDefault(::testing::ReturnRef(mem));
-        ON_CALL(context, get_parent_cd_size).WillByDefault(::testing::Return(data.size()));
-        ON_CALL(context, get_parent_cd_addr).WillByDefault(::testing::Return(0));
-        ON_CALL(context, get_last_rd_addr).WillByDefault(::testing::Return(0));
-        ON_CALL(context, get_last_rd_size).WillByDefault(::testing::Return(data.size()));
+        EXPECT_CALL(context, get_memory).WillRepeatedly(::testing::ReturnRef(mem));
```

Comment:
Numerous changes from `ON_CALL -> EXPECT_CALL` because of the change from `NiceMock` to `StrictMock`